src/store/index.ts

```diff
@@ -4,20 +4,28 @@
  * @see https://vuex.vuejs.org/guide/structure.html
  */
 
-import { createStore } from 'vuex';
+import { createStore, Store } from 'vuex';
 
 import actions from './actions';
 import mutations from './mutations';
 import RootState from './RootState';
 
-export default createStore( {
-	state(): RootState {
-		return {
-			lemma: '',
-			language: '',
-			lexicalCategory: '',
-		};
-	},
-	mutations,
-	actions,
-} );
+type StoreParams = { licenseUrl?: string, licenseName?: string };
+
+export default function initStore( { licenseUrl = '', licenseName = '' }: StoreParams ): Store<RootState> {
```

Comment:
Editorial note: this `initStore()` refactoring was extracted into #51 and already merged.

Reply:
Hmm. But it doesn't include the license text and url yet does it?

Reply:
Nope, just the change from exporting a store instance to a function.